pages/basic_concepts/promotion.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Promotion policies
+last_updated: January 11, 2023
+toc: false
+permalink: promotion.html
+---
+
+## What is promotion policy
+Promotion policy is a "filter" working on per-request basis that determines
+what gets inserted(promoted) into the cache. This filter is meant to limit cache
+pollution resulting from insertion of rarely accessed data (e.g. I/Os resulting
+from Ceph scrub operation). If promotion policy rejects a request it is
+handled in pass-through, similarly to what happens when request couldn't be inserted
+into cache because of other reasons (e.g. unable to evict enough cache lines).
+
+Currently OCF implements only **nhit** promotion policy.
+
+## Nhit promotion policy
+### Description
+Nhit promotion policy works by tracking accesses to core lines and allowing cache
+insertion for those, which were accessed `insertion_threshold` times. Nhit policy
+will let all requests through and won't count any accesses until cache occupancy
+reaches `trigger_threshold`%.
```

Comment:
I'd like a slightly more detailed description of the purpose of the trigger_threshold parameter

Reply:
added a new section describing parameters, what do you think?